dynamicio/config.py

```diff
@@ -83,15 +83,16 @@ def with_module(cls, module: ModuleType):
         """
         return type(f"{cls.__name__}_{module.__name__}", (cls,), {"module": module})
 
-    def dyn_str_constructor(self, node: yaml.nodes.ScalarNode) -> str:
-        """Responsible for the switching of one or more "[[ DYNAMIC_VAR ]]" strings with the respective attributes value in a give module.
+    def dyn_str_constructor(self, node: yaml.nodes.ScalarNode, is_schema: bool = False) -> Any:
```

Comment:
`nit` :why not use 
```str | float```
instead of `Any` for the type hint?

Reply:
Done

Reply:
Actually, mypy takes issue with it. Leaving it as is.

Reply:
Oh my bad it must be because you're not on python 3.10, then the right way to write unions is: `Union[str, float]`